src/nodedef.cpp

```diff
@@ -112,6 +183,71 @@ void NodeBox::deSerialize(std::istream &is)
 		wall_side.MinEdge = readV3F1000(is);
 		wall_side.MaxEdge = readV3F1000(is);
 	}
+	else if (type == NODEBOX_CONNECTED)
+	{
+		u16 count = readU16(is);
+		fixed.reserve(count);
+		while (count--) {
+			aabb3f box;
```

Comment:
Yeah, it *has* to be an ordering problem.

The best I can do is this:

```
-               while (count--)
-                       connect_up.push_back(aabb3f(readV3F1000(is), readV3F1000(is)));
+               while (count--) {
+                       v3f min = readV3F1000(is);
+                       v3f max = readV3F1000(is);
+                       connect_up.push_back(aabb3f(min, max));
+               }
```

This way, we're doing minimal work prior to calling the constructor. There's no way around the ordering failure that I can see otherwise.